schedulearn-frontend/src/server-components/Views/TopicsByManagerView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ManagedTeamsSelect from "../ManagedTeamsSelect";
+import FlatTopicList, { FlatTopicListItem } from "src/components/FlatTopicList/FlatTopicList";
+import UserContext from "src/UserContext";
+
+interface TopicsByManagerViewState {
+  topics?: FlatTopicListItem[];
+}
+
+export default class TopicsByManagerView extends React.Component<{}, TopicsByManagerViewState> {
+  state: TopicsByManagerViewState = {}
+
+  handleTeamClick = (teamId: number): void => {
+    UserContext
+      .fetch(`api/Topic/team/${teamId}`)
+      .then((response) => {
+        if (!response.ok) {
+          //set error
+          return;
+        }
+
+        return response.json();
+      })
+      .then((teamsTopics: FlatTopicListItem[]) => {
+        this.setState({ topics: teamsTopics });
+      });
+    console.log("Paspaude " + teamId);
+  }
+
+  render(): React.ReactNode {
+    const teamTopics = this.state.topics !== undefined ?
+      (<FlatTopicList topics={this.state.topics} />) : undefined;
```

Comment:
Perhaps it would be better to just pass an empty list, as such (this.state.topics ?? [])